dist/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Restaurant Page</title>
+</head>
+
+<body>
+  <nav class="nav-wrapper">
```

Comment:
We should have only 3 tags in `index.html`.
![image](https://user-images.githubusercontent.com/31897434/76575132-9c799900-64be-11ea-8eea-ebf09181b298.png)